### PR TITLE
Add hypnotoad

### PR DESCRIPTION
**Pull request by johnomotani**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

Pinging @ZedThree, could you post a comment to confirm you're happy to be a maintainer please?

### CODE REVIEW

File: recipes/hypnotoad/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,53 @@
+{% set name = "hypnotoad" %}
+{% set version = "0.1.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256:
```
Comment (synapticarbors) [inline recipes/hypnotoad/meta.yaml:10]: You're going to need the hash. You can get it off of the pypi website:

https://pypi.org/project/hypnotoad/#files

File: recipes/hypnotoad/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
```
Comment (synapticarbors) [inline recipes/hypnotoad/meta.yaml:15]: It looks like this package has python console script entry points. Please add them:

https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#python-entry-points

and test them under `commands`:

https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#test-commands

**Comment by ZedThree** [comment]
Happy to be a maintainer!

**Comment by johnomotani** [comment]
@conda-forge/help-python can somebody help me debug this failure please? A test of importing `Qt.QtWidget` is failing with the error
```
ModuleNotFoundError: No module named 'Qt.QtWidgets'; 'Qt' is not a package
```
`Qt` is provided by `qt.py`, which I even added to the `test:requires` to be doubly sure it should be available. Import of `Qt` succeeds, and on my system `conda build .` succeeds in `recipes/hypnotoad`.

I'm trying to check the import of `Qt.QtWidgets` because the failure of that import was causing a failure of the check for existence of my `hypnotoad-gui -h` command.

**Comment by nehaljwani** [comment]
>Qt.py enables you to write software that runs on any of the 4 supported bindings - PySide2, PyQt5, PySide and PyQt4.

Sounds like you'll need one of these in your `test:requires` as well!

**Comment by johnomotani** [comment]
Thanks @nehaljwani. I think `PyQt5` was being installed already, but I added it explicitly to be sure, and checked that it imports OK, but still get the same error with `Qt.QtWidgets`. Any other ideas?

**Comment by nehaljwani** [comment]
Qt has a runtime requirement on `/usr/lib64/libGL.so`. 
```
$ python -c 'import Qt.QtWidgets'
$ sudo mv /usr/lib64/libGL.so.1.2.0{,.bak}
$ python -c 'import Qt.QtWidgets'
Traceback (most recent call last):
  File "<string>", line 1, in <module>
ModuleNotFoundError: No module named 'Qt.QtWidgets'; 'Qt' is not a package
```

You'll need to add `mesa-libGL` to yum_requirements.txt. xref: https://conda-forge.org/docs/maintainer/knowledge_base.html#yum-deps

**Comment by johnomotani** [comment]
Thanks for the help @nehaljwani! I think this PR is ready for review now.

**Comment by johnomotani** [comment]
@conda-forge/help-python this PR is ready for review/merge. Thanks!

**Comment by nehaljwani** [comment]
Thank you @johnomotani !

**Pull request merged**